### DISCUSSION TITLE
Remote Station

### DISCUSSION THREAD

**Discussion by hjerald1** [dl9rdz/rdz_ttgo_sonde#405] [TTGO hardware]
I have a remote hilltop solar setup with a wireless point-to-point conection to my home router in the valley. (https://community.ui.com/stories/A-Remote-OTA-TV-Antenna-for-Streaming-Local-TV-Stations-With-a-Ubiquiti-PtP-Bridge-TV-WiFi-Tuner-an/f56ec936-3029-4811-a7bb-2f76d35b198d).

I would like to add rdz_ttgo_sonde hardware to the hilltop setup. Is there a way to connect the ttgo board directy to the network through an available RJ45 port, thereby transmitting the ttgo data from the hilltop through the network to a PC at my home?

**Comment by dl9rdz** [comment]
Currently there is no such option.  There might be one (in the far future) with an external SPI/RJ45 module like the W5500, but likely not in the default binary image (i.e. you will have to compile your own firmware with that support added)

You could also simple place a tiny RJ45/WiFi gateway, a tiny and cheap thing like a TL-WR802N ar hAP lite/mAP lite would do.

**Comment by hjerald1** [comment]
Thanks much for your input. Your suggestions got me to thinking. The Ubiquiti SolarPoint charge conroller in my setup has a built in WiFi management radio. So I will just turn that feature back on as see if it that will connect the TTGO into the network. We will see.
Thanks again.
